conference/react-js/src/components/AddNewOrganizer/AddNewOrganizer.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Dialog from 'material-ui/Dialog';
+import RaisedButton from 'material-ui/RaisedButton';
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+import TextField from 'material-ui/TextField';
+import { conferencesNames } from '../../constants/backend-url';
+
+class AddNewOrganizer extends Component {
+  constructor(prop) {
+    super(prop);
+    this.state = {
+      open: false,
+      value: false,
+      conferenceList: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get(conferencesNames)
+      .then(({ data }) => {
+        this.setState({ conferenceList: data });
+      });
+  }
+
+  setMenuItems = () => (
+    this.state.conferenceList.map(name => (
+      <MenuItem
+        value={name}
+        primaryText={name}
+        key={name}
+      />),
+    )
+  )
+
+  handleChange = (event, index, value) => this.setState({ value });
+
+  toggleInput = () => {
+    this.setState({ open: !this.state.open });
```

Comment:
I think callback will be better, when we are based on previous state. `({ open }) => ({ open: !open })`